<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreInfoBoard</class>
 <widget class="QWidget" name="ScoreInfoBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="4">
      <widget class="QLabel" name="label_ShiftPreferences_Result">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Shift preferences</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="14" column="0">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="11" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_ShiftPreferences_Number">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>=</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_ShiftPreferences_Value">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>x</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLabel" name="label_42">
       <property name="text">
        <string>x</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="5">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Dayoff preferences</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>=</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="5">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Unassigned shifts</string>
       </property>
      </widget>
     </item>
     <item row="19" column="3">
      <widget class="QLabel" name="label_40">
       <property name="text">
        <string>=</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_DayoffPreferences_Value">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="0" colspan="5">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Long rests</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="5">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Early after late shifts</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="5">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Consecutive late shifts</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0" colspan="5">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Deviation target late shifts</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_DayoffPreferences_Number">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_LongRests_Value">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_UnassignedShifts_Value">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>x</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>=</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_DayoffPreferences_Result">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>x</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_UnassignedShifts_Number">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>=</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QLabel" name="label_UnassignedShifts_Result">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="19" column="4">
      <widget class="QLabel" name="label_DeviationTargetLateShifts_Result">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="4">
      <widget class="QLabel" name="label_LongRests_Result">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_EarlyAfterLateShifts_Value">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>x</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="3">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>=</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="3">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>=</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QLabel" name="label_LongRests_Number">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>x</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QLabel" name="label_EarlyAfterLateShifts_Number">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="4">
      <widget class="QLabel" name="label_EarlyAfterLateShifts_Result">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_ConsecutiveLateShifts_Value">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>x</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QLabel" name="label_ConsecutiveLateShifts_Number">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="4">
      <widget class="QLabel" name="label_ConsecutiveLateShifts_Result">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_DeviationTargetLateShifts_Value">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="19" column="2">
      <widget class="QLabel" name="label_DeviationTargetLateShifts_Number">
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
